### ISSUE TITLE
Javascript Generated websites

### ISSUE THREAD

**Issue by L11K** [huginn/huginn#2349]
I tried to use website agent with https://www.mobirum.com/en/knightschronicle, but the output keep returning empty, so I came here and searched issues/wiki, found [Browser Emulation with PhatomJS](https://github.com/huginn/huginn/wiki/Browser-Emulation-Using-PhantomJs-Cloud)

I removed the first part since I don't have rss to give. I did something wrong, because it doesn't seem to work. [Scenario](https://pastebin.com/TYBtQa4q)

Thank you in advance.

**Comment by dsander** [comment]
You need to set `url_from_event` to `{{url}}`, but sadly it looks like the phantomjscloud service does not extract the website properly. If you open the `url` of the Event created by the PhantomJsCloudAgent in your browser it will not contain much HTML.

**Comment by L11K** [comment]
Yeah, it doesn't. So bothersome, I have no Idea what I could do with my limited skills.

**Comment by dsander** [comment]
https://www.browserless.io/ seems to be working well, their code is open source and they have a docker image: https://github.com/joelgriffith/browserless#-docker-quickstart

```
curl -X POST \
  http://localhost:3000/content \
  -H 'Cache-Control: no-cache' \
  -H 'Content-Type: application/json' \
  -d '
{
  "url": "http://www.mobirum.com/en/knightschronicle"
}'| pup ".c_atc_tt:nth-child(1) strong"

<strong>
 <span>
  [Notice]
 </span>
 Novice Knights Guide - 3
</strong>
<strong>
 <span>
  [Notice]
 </span>
 Arena Recommended and Banned Heroes List (2nd week of August)
</strong>
<strong>
 <span>
  [Notice]
 </span>
 KnightsGram #3
</strong>
<strong>
.....

```

**Comment by L11K** [comment]
I'm the free-ish Heroku to host huginn, as it was the best I could find for this.  I'm not sure If I will be able to use browserless together with huginnI(Both by lacking how-to and Heroku itself)

I will try to read on it. Appreciate your help.

**Comment by nageshuizeii** [comment]
Great, this completely solved my original problem, I just need to change the URL to https://localhost:3000/{{url}} in the website agent. I immediately tried it with docker, which is much better than PhatomJS.
